.github/workflows/ci.yml

```
@@ -0,0 +1,38 @@
+name: Continuous Integration
+
+on:
+  push:
+
+jobs:
+  build-and-publish:
+    name: Build and Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Run tests
+        run: docker-compose -f action-services.yml run portal /data/run-tests.sh
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
+          tags: |
+            type=ref,event=branch
```

Do you want to add `,enable=${{ github.ref_name != 'main' }}` here? I think that would make this equivalent to what we had in Codeship (i.e. not pushing a 'main' tag).

**WARNING:** I have not verified the syntax on this, just copied and tweaked from line 30.

```suggestion
            type=ref,event=branch,enable=${{ github.ref_name != 'main' }}
```

---

I'm fine with this.